### ISSUE TITLE
[BUG] - `pre` elements in Storefront use wrong focus color

### ISSUE THREAD

**Issue by anotherMatze** [porsche-design-system/porsche-design-system#2343]
### JS Framework

Storefront

### Porsche Design System version

latest

### Reproduction Example

https://designsystem.porsche.com/latest/news/changelog/components

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/latest/news/changelog/components
2. Scroll down to the first occurrence of a displayed code diff
3. Focus it by keyboard navigation
4. See wrong focus color

### Expected behavior

Code diffs should be accessible via keyboard and use correct focus color provided by Styles.

**Comment by michaelworm** [comment]
https://github.com/porsche-design-system/porsche-design-system/pull/2947
